python-sdk/src/astro/databases/base.py

```diff
@@ -774,3 +774,9 @@ def check_file_pattern_based_schema_autodetection_is_supported(self, source_file
             source_file.location.location_type in self.FILE_PATTERN_BASED_AUTODETECT_SCHEMA_SUPPORTED
         )
         return is_file_pattern_based_schema_autodetection_supported
+
+    def openlineage_dataset_name(self, table: BaseTable):
+        raise NotImplementedError
```

Comment:
added `table` param to get the dataset and table name in case of bigquery. Airflow biquery hook method `hook.get_uri()` return string which has only project id and scheme f"bigquery://{self.project_id}".